cert_agent/defaults/main.yml

```
@@ -28,6 +28,9 @@ cert_agent_statsd_host: 'stage-prometheus.infra.appsembler.com'
 cert_agent_statsd_port: 9125
 cert_agent_statsd_prefix: 'tahoe_cert_agent'
 
+cert_agent_honeycomb_dataset: "changeme"
+cert_agent_honeycomb_writekey: "changeme"
```

This will break if we don't set it, right?

```suggestion
cert_agent_honeycomb_dataset: !!null
cert_agent_honeycomb_writekey: !!null
```